simphony/scripts/cuba_generate.py

```diff
@@ -84,7 +84,8 @@ def table(input, output):
         'uuid': 'String',
         'string': 'String',
         'double': 'Float64',
-        'integer': 'Int32'}
+        'integer': 'Int32',
```

Comment:
The suggested change in https://github.com/simphony/simphony-common/pull/253/files#r47629563 would help remove the MaterialRelation-related stuff here.